R/FilterStates.R

```diff
@@ -11,7 +11,7 @@
 #' This expression is hereafter referred to as \emph{subset expression}.
 #'
 #' The \emph{subset expression} is constructed differently for different
-#' classes of the underlying data object and `FilterStates` subclasses.
+#' classes of the underlying data object and `InteractiveFilterStates` subclasses.
```

Comment:
it's initialized in the different way depending on the parent class (FilterStates) 
```suggestion
#' classes of the underlying data object and `FilterStates` subclasses.
```